src/content/contactForm/SmallTextField.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import styles from "../../style/ContactForm.css";
+import classNames from "classnames";
+import React, { Component, Fragment} from "react";
+import {InputField} from "../../contacts";
```

Comment:
why are you separeted final implementation but left InputField in contacts.js

Reply:
done